README.md

```diff
@@ -50,21 +41,58 @@ _How can I build and deploy it (lots of this will be links out as the steps will
 * CI provided by CircleCI: [concordances-rw-dynamodb](https://circleci.com/gh/Financial-Times/concordances-rw-dynamodb)
 
 ## Service endpoints
-_What are the endpoints offered by the service_
+See the api/api.yml for the swagger definitions of these endpoints
 
-e.g.
 ### GET
 
-Using curl:
-
-    curl http://localhost:8080/_<INSERT SEPCIFIC URL HERE>_| json_pp`
-
-_Explain what the response should represent_
-
-Based on the following [google doc](_<INSERT API DOCUMETATION HERE>_)
+Using curl:  
+  _request:_
```

Comment:
can we have descriptions of each of the endpoints not just the examples. I suspect in time these request examples will be removed in favour of the swagger files but we are not quite at that point yet